Pod/Classes/UITableView+Extension.swift

```diff
@@ -80,7 +80,11 @@ extension UITableView: UIGestureRecognizerDelegate {
   }
   
   public func gestureRecognizer(gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWithGestureRecognizer otherGestureRecognizer: UIGestureRecognizer) -> Bool {
-    return true
+    if gestureRecognizer == revealPanGesture || otherGestureRecognizer == revealPanGesture {
```

Comment:
I prefer to use a ternary operator, otherwise this looks great. Will get it merged asap.
Apologies for delays on this, I just realised I have 3 PRs on this project that I wasn't notified of.

Thanks.